### ISSUE TITLE
Dockerize.

### ISSUE THREAD

**Issue by ChristopherLandaverde** [ChristopherLandaverde/Lixt#51]
Current Issue: My current attempts at dockerize my app has led me to create an image that gives me a 

" (HTTP - Code 400) unexpected - invalid reference format " 

I'm never able to successfully to run a container. If anyone can assist in dockerize this app or have a better way to accomplish this for a react-native project, that would be helpful.

**Comment by dhairya137** [comment]
Can J dockerize this app?

**Comment by ChristopherLandaverde** [comment]
Yeah, that would be awesome!. Sorry for the late response, was fixing some bugs. Thank you again!

**Comment by dhairya137** [comment]
I will dockerize this app

**Comment by ChristopherLandaverde** [comment]
I can't send any information towards the API when I dockerized the app, I can only get it running locally. In my most recent attempt the image will be created, but no the container. 
I will add more documentation.
